docs/csharp/misc/CS4009.md

```diff
@@ -14,7 +14,7 @@ helpviewer_keywords:
 You cannot use the `async` keyword in the application entry point (typically the `Main` method).
 
 > [!IMPORTANT]
-> Starting with C# 7.1, the `Main` method can have an `async` modifier. For more information, see the [Async main](../whats-new/csharp-7.md#async-main) section of the [What's new in C# 7](../whats-new/csharp-7.md) article. For information about how to select the C# language version, see the [Select the C# language version](../language-reference/configure-language-version.md) article.
+> Starting with C# 7.1, the `Main` method can have an `async` modifier. For more information, see the [Async main](../whats-new/csharp-version-history.md#c-version-70) section of the C# Version history article. For information about how to select the C# language version, see the [Select the C# language version](../language-reference/configure-language-version.md) article.
```

Comment:
```suggestion
> Starting with C# 7.1, the `Main` method can have an `async` modifier. For more information, see [Async main return values](../fundamentals/program-structure/main-command-line.md#async-main-return-values). For information about how to select the C# language version, see the [Select the C# language version](../language-reference/configure-language-version.md) article.
```

Reply:
Or maybe this and the other async main links should point to https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.1/async-main?branch=pr-en-us-27112?